ccbc-worker/src/test/java/io/coin/ccbc/worker/infra/exchange/CoinOneHttpClientHttpApiRealTest.java

```diff
@@ -0,0 +1,76 @@
+package io.coin.ccbc.worker.infra.exchange;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.coin.ccbc.worker.infra.exchange.config.HttpClientConfig;
+import io.coin.ccbc.worker.infra.exchange.dto.CoinOneTickerDto;
+import java.io.IOException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import retrofit2.Response;
+
+/**
+ * @author Bomi
+ * @date 2021/07/22
+ */
+
+class CoinOneHttpClientHttpApiRealTest {
```

Comment:
```suggestion
class CoinOneHttpClientHttpApisTest {
```
class 이름은 테스트하려는 클래스에 Test 붙인 이름이면 깔끔할 것 같아요~!